manifests/n/NoMachine/NoMachineClient/7.4.1/NoMachine.NoMachineClient.yaml

```
@@ -1,21 +1,21 @@
 PackageIdentifier: NoMachine.NoMachineClient
 Publisher: NoMachine S.a.r.l.
-PackageName: NoMachine Enterprise
+PackageName: NoMachine Entreprise Client
```

```suggestion
PackageName: NoMachine Entreprise Client
```
`Enterprise` is misspelled here.

---

Fixed